templates/editor/boot-partial.html

```diff
@@ -14,5 +14,15 @@
         }
       };
     {% endif %}
+
+    if (!window.onPiskelReady) {
+      window.onPiskelReady = [];
+    }
+
+    window.onPiskelReady.push(function() {
```

Comment:
It's not mandatory here given the implementation, but it makes the pattern harder to reapply somewhere else without modifications and requires the reader to know that boot-partial is executed first. 

We can either:
- change the implementation (if piskel already loaded -> callback(), otherwise put the callback in the array ...) 
- or add a comment explaining why we can safely push to the array here